iree/vm/c_funcs.h

```diff
@@ -17,8 +17,21 @@
 
 #include <stdint.h>
 
-// Arithmetic ops
-inline int32_t vm_add_i32(int32_t a, int32_t b) { return a + b; }
+//===------------------------------------------------------------------===//
+// Native integer arithmetic
+//===------------------------------------------------------------------===//
+
+int32_t vm_add_i32(int32_t lhs, int32_t rhs) { return lhs + rhs; }
```

Comment:
these should be `static inline` (otherwise linkers will start complaining about ODR violations and such)

Reply:
Thanks for the hint :) Changed it accordingly.